packages/framework/src/Component/Domain/Config/DomainConfig.php

```diff
@@ -62,6 +62,14 @@ public function getLocale()
         return $this->locale;
     }
 
+    /**
+     * @return string
+     */
+    public function getLocaleUppercase(): string
```

Comment:
yeah, I was also not sure about it, I wanted to keep the swift pattern with uppercase, as well as the original payment names in data fixtures, but the truth is, it might bring more troubles than benefits, removed in c60eb0882a